program/cylindrically-rotate-a-matrix-by-90-degrees/CylindricallyRotateAMatrixBy90Degrees.ts

```diff
@@ -0,0 +1,15 @@
+const rotateClockwise = (matrix: number[]): number[] => {
```

Comment:
you have an incorrect type for the parameter and the output as well.

Reply:
@victoriacheng15  Its a numeric array , so we should use the number[] for the input & output data type .